# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
geometry_msgs/PoseStamped[] poses
float32 force_thresh
bool ignore_ft
